### PR TITLE
Hot fix flags and targets

### PR DESCRIPTION
**Pull request by kaitejohnson**
@dylanhmorris I realized a few silly errors in some of the changes from #120. I forgot to add the location to the `ww_data_flags` table and combine outputs is set up to do just a straight rowbind, so it doesn't pass along location metadata by default. 

Also realized a small bug in the name of the file I was looking for when grabbing the wastewater quantiles.

### CODE REVIEW

File: wweval/R/get_table_sufficient_ww.R [hunk 1]
```diff
@@ -81,6 +85,7 @@ get_ww_data_flags <- function(input_ww_data,
       mean_log_ww = mean(log(ww))
     ) |>
     dplyr::mutate(
+      location = this_location,
```
Comment (dylanhmorris) [inline wweval/R/get_table_sufficient_ww.R]:
```suggestion
      location = !!this_location,
```

**Review by dylanhmorris — COMMENTED** [review]
One key change

**Comment by kaitejohnson** [comment]
Whoops I also used `select` instead of `distinct`

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**